### ISSUE TITLE
Feature request - Customize icons according to file type

### ISSUE THREAD

**Issue by esibruti** [files-community/Files#5031]
It would be interesting to add the possibility that we can change the icon according to the type of file. It would be an option located in the settings -> personalization. There would be an option of the types of files and the icons currently associated. It would be an implementation of the FilesTypeMan application but just so that we can customize the icons. If you get confused, write in the comments that I will explain in another way.

**Comment by esibruti** [comment]
@yair100 No, no, I didn't even know about this application ... but that's not it. If you search for FilesTypeMan, this is an application that detects all types of files that the computer recognizes and with that, he can change the icon for the entire computer. It would be interesting to do that but in a smaller format, in the sense that it is just a customization of the file icon. For example, I use Photoshop CS6 and the computer does not detect the application as installed because it is portable. Because of this, the default white icon appears in the psd files. And so the idea was to introduce something like that to Files.

**Comment by yaichenbaum** [comment]
It's an interesting idea, it's a little out of scope to build this kind of functionality into Files, but I'll keep this open to gather feedback and to see if we can implement this a different way.

**Comment by TizianoMad** [comment]
I understand the issue. For example, to display "PDF" icon instead of the Chrome logo, I had to tweak Chrome's behavior in its reg files ans add the .ico file in its system folder. Obviously, it would have been easier to change the rule directly in file explorer settings.

**Comment by Josh65-2201** [comment]
Asking @yair100 for approval.
